### ISSUE TITLE
[Due for payment 2026-02-23] Screen Reader: Landing Page: The combobox of Language has no programmatic name

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74855]
## Action Performed
1. Using JAWS + Chrome, open the page https://new.expensify.com/
2. Press Tab key to navigate to the language combo box, located in the footer under 'Get Started'
3. Observe the announcement

## Expected Result
The Language combo box will be announced with an accessible name and the currently selected value. For example:
-"Language selection, Combo box, English"

## Actual Result
The language selection combo box does not announce with an accessible name. When focused, the element only announces as 'combo box' with the selected value. For example:
--"Combo box collapsed English"

HTML:
<select tabindex="-1" data-testid="web_picker" style="font-family: &quot;Expensify Neue&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 400; font-size: 11px; padding: 6px 17px 6px 0px; border-width: 0px; color: rgb(231, 236, 233); appearance: none; height: 26px; opacity: 1; background-color: rgb(0, 46, 34); cursor: pointer;"><option value="de" label="Deutsch" style="color: rgb(231, 236, 233);">Deutsch</option><option value="en" label="English" style="color: rgb(231, 236, 233);">English</option><option value="es" label="Español" style="color: rgb(231, 236, 233);">Español</option><option value="fr" label="Français" style="color: rgb(231, 236, 233);">Français</option><option value="it" label="Italiano" style="color: rgb(231, 236, 233);">Italiano</option><option value="nl" label="Nederlands" style="color: rgb(231, 236, 233);">Nederlands</option><option value="pl" label="Polski" style="color: rgb(231, 236, 233);">Polski</option><option value="pt-BR" label="Português&nbsp;(BR)" style="color: rgb(231, 236, 233);">Português&nbsp;(BR)</option><option value="zh-hans" label="中文&nbsp;(简体)" style="color: rgb(231, 236, 233);">中文&nbsp;(简体)</option><option value="ja" label="日本語" style="color: rgb(231, 236, 233);">日本語</option></select>

## Area issue was found
Landing Page

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Use aria-label to give the element an accessible name, such as 'Language selection'
https://www.w3.org/WAI/ARIA/apg/patterns/combobox/

## Environment(s)
Windows Windows 10 - Chrome

## Screenshots/Videos
<img width="1916" height="997" alt="Image" src="https://github.com/user-attachments/assets/4dee2c65-c4b2-4a1a-b1d1-17c49927e62b" />

https://github.com/user-attachments/assets/7488026f-4294-44d2-8855-08cd5e465938

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 - NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6995218
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by rushatgabhane** [comment]
fixed https://github.com/Expensify/App/issues/79233
